desktop/ui/src/main/java/org/datacleaner/windows/AboutDialog.java

```diff
@@ -107,7 +107,7 @@ protected int getDialogWidth() {
 
     @Override
     protected boolean isWindowResizable() {
-        return true;
+        return false;
```

Comment:
Hmmm... Okay, okay. The license reading pretty much makes it something you might want to resize. Dangit. I hate even having chrome on about dialogs.

Oh well.